src/igvfd/schemas/changelogs/page.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`page.json`*
 
+### Schema version 3
+
+* Update schema version for deletion of unused properties
```

Comment:
Can yo please follow the following style of changelog updates: 
* Remove `news`, `some_other_property`.
* Update `layout` to include `some_addition`...